session_1/hello_world/app/views/page/root.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>welcome to Rails</h1>
```

Comment:
@harishA3 
Our test expects you to display `Hello, World!`, but you are displaying `Welcome to Rails`
This doesn't matter as you have understood how to display data using HTML tags.